Swift/Controllers/UIInterfaces/XMLConsoleWidget.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <Swiften/Base/SafeByteArray.h>
+#include <boost/signals2.hpp>
```

Comment:
In Swift we aim to keep our includes sorted. There's a handy script for this which aims to do the job for most cases, https://github.com/swift/swift/blob/master/BuildTools/FixIncludes.py . Simply run it like `./BuildTools/FixIncludes.py -i path_to_cpp_or_h_file.cpp`.